cupy/cuda/compiler.py

```diff
@@ -263,10 +323,29 @@ def _compile_with_cache_cuda(source, options, arch, cache_dir,
         ptx = compile_using_nvrtc(source, options, arch, name + '.cu')
         ls = function.LinkState()
         ls.add_ptr_data(ptx, 'cupy.ptx')
+        # for separate compilation
+        if _check_cudadevrt_needed(options):
+            global _win32, _osx
+            # defer import to here to avoid circular dependency
+            from cupy.cuda import get_cuda_path
+            cudadevrt = get_cuda_path()
+            if _win32:
+                # rely on os.altsep
+                cudadevrt += '/lib/x64/cudadevrt.lib'
+            elif _osx:
+                cudadevrt += '/lib/libcudadevrt.a'
+            else:  # linux
+                cudadevrt += '/lib64/libcudadevrt.a'
```

Comment:
Use `lib64` first, and if it does not exist, use `lib`.
c.f. https://github.com/cupy/cupy/blob/b9db374f4d0ccee358a998d7b283248f77cef351/install/build.py#L115

Reply:
Ah OK. Thanks. I wasn't aware that `lib64` is possible on Mac.

Reply:
I guess I parsed your comment wrong. Now I fixed it.